components/avatar/appearance.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Appearance
+page_title: Avatar Appearance
+description: Appearance settings of the Avatar for Blazor.
+slug: avatar-appearance
+tags: telerik,blazor,avatar,appearance
+published: True
+position: 10
+---
+
+# Appearance Settings
+
+You can control the appearance of the Avatar by setting the following attribute:
+
+* [Size](#size)
+* [Rounded](#rounded)
+* [FillMode](#fillmode)
+* [ThemeColor](#themecolor)
+
+You can use all of them together to achieve the desired appearance. This article will explain their effect one by one.
+
+## Size
+
+You can increase or decrease the size of the Avatar by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Avatar.Size` class:
+
+| Class members | Manual declarations |
+|---------------|--------|
+| `Small`   |`sm`|
+| `Medium` (default value)  |`md`|
+| `Large`   |`lg`|
+
+> The default value of the `Size` will take precedence over the values of the `Width` and `Height` parameters. Set the `Size` to an empty `string` to apply your custom `Width` and `Height`.
+
+>caption The built-in Avatar sizes
+
+````CSHTML
+@{
+    var fields = typeof(Telerik.Blazor.ThemeConstants.Avatar.Size)
+        .GetFields(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Static
+        | System.Reflection.BindingFlags.FlattenHierarchy)
+        .Where(field => field.IsLiteral && !field.IsInitOnly).ToList();
+
+    foreach (var field in fields)
+    {
+        string size = field.GetValue(null).ToString();
+
+        <div style="float:left; margin: 20px;">
+        <TelerikAvatar Size="@size" Type="AvatarType.Text">
+            JD
+        </TelerikAvatar>
+        </div>
+    }
+}
+````
+
+## Rounded
+
+The `Rounded` parameter applies the `border-radius` CSS rule to the Avatar to achieve curving of the edges. You can set it to a member of the `Telerik.Blazor.ThemeConstants.Avatar.Rounded` class:
+
+| Class members | Manual declarations |
+|------------|--------|
+|`Small` |`sm`|
+|`Medium`|`md`|
+|`Large`|`lg`|
+|`Full` (default value) |`full`|
+
+>caption The built-in values of the Rounded attribute
+
+````CSHTML
+@* The built-in rounded options of the Avatar.  *@
+
+@{
+    var fields = typeof(Telerik.Blazor.ThemeConstants.Avatar.Rounded)
+        .GetFields(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Static
+        | System.Reflection.BindingFlags.FlattenHierarchy)
+        .Where(field => field.IsLiteral && !field.IsInitOnly).ToList();
+
+    foreach (var field in fields)
+    {
+        string rounded = field.GetValue(null).ToString();
+
+        <div style="float:left; margin: 20px;">
+            <TelerikAvatar Rounded="@rounded" Type="AvatarType.Text">
+                JD
+            </TelerikAvatar>
+        </div>
+    }
+}
+````
+
+## FillMode
+
+The `FillMode` controls how the TelerikAvatar is filled. You can set it to a member of the `Telerik.Blazor.ThemeConstants.Avatar.FillMode` class:
```

Comment:
We are using the word "fill" to explain "fillmode". It's not that self-evident. Try something else, e.g. "background", etc.